include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Yes, the problem is... it is quite likely that someone didn't define USE_LONG_LONG and it is much less likely that someone writes long long code on a system where it isn't supported.

But that said, related to the above error. Can you add a constructor to the struct, so that you get the runtime error?